plugins/mc.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: mc
+spec:
+  version: v0.5.0
+  homepage: https://github.com/jonnylangefeld/kubectl-mc
+  shortDescription: Run kubectl commands against multiple clusters at once
+  description: |+2
+    This plugin helps to work with multiple kubernetes clusters at once, filtered by
+    a regular expression. For instance to get the status of a deployment across all
+    `staging` clusters run the following command:
+    mc -r staging -- get deployments -n kube-system
+
+    For more examples run kubectl mc --help
```

Comment:
this line might also be redundant (I assume `kubectl mc` already shows help).

we also tell users how to use this plugin after installation:

```
Installed plugin: tree
\
 | Use this plugin:
 | 	kubectl tree
 | Documentation:
 | 	https://github.com/ahmetb/kubectl-tree
[...]
```

Reply:
you're right it does! I'll remove this line